db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -950,21 +950,31 @@ Status DBImpl::IncreaseFullHistoryTsLow(ColumnFamilyHandle* column_family,
 Status DBImpl::IncreaseFullHistoryTsLowImpl(ColumnFamilyData* cfd,
                                             std::string ts_low) {
   VersionEdit edit;
+  std::string current_ts_low;
   edit.SetColumnFamily(cfd->GetID());
   edit.SetFullHistoryTsLow(ts_low);
 
   InstrumentedMutexLock l(&mutex_);
-  std::string current_ts_low = cfd->GetFullHistoryTsLow();
+  current_ts_low = cfd->GetFullHistoryTsLow();
```

Comment:
Why separate declaration and definition?

Reply:
Thank you for catching this. It's from a previous version of change, I forgot to put it back. It's reverted now.